app/assets/javascripts/oxalis/model/actions/skeletontracing_actions.js

```diff
@@ -394,3 +369,36 @@ export const setTreeGroupAction = (groupId: ?string, treeId: number): SetTreeGro
   groupId,
   treeId,
 });
+
+// The following actions have the prefix "AsUser" which means that they
```

Comment:
Nice, do you think moving the `deleteTreeWithConfirmAction` here as well would be sensible?

Reply:
Yes, good point, I'll do that!